### DISCUSSION TITLE
is it free?

### DISCUSSION THREAD

**Discussion by PtDragon** [gravitl/netmaker#2364] [Q&A] [answered]
Hello! 
Is it free and fully self-hosted or paid?
Or maybe it have both options like ZeroTier(it is free if you self-host controller but if you use their website it's limited or paid)?
I'm new here so I want to know about it before trying to install.

**Comment by mattkasun** [comment, marked as answer]
The CE edition is completely free and self hosted. the EE version is also self hosted and has both free and paid tiers.  The SaaS version has free and paid tiers.
